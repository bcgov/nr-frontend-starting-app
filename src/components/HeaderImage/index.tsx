import styled from 'styled-components';

const HeaderImage = styled.img`
  @media screen and (max-width: 640px) {
    margin-top: 10px;
    height:30px;
  }
`;

export default HeaderImage;
