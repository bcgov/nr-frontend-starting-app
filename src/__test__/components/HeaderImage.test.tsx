/* eslint-disable no-undef */
import { render } from '@testing-library/react';
import React from 'react';
import renderer from 'react-test-renderer';
import HeaderImage from '../../components/HeaderImage';
import '@testing-library/jest-dom';

describe('the component', () => {
  it('should have the correct image', () => {
    const { getByTestId } = render(<HeaderImage data-testid="header-image" src="https://developer.gov.bc.ca/static/BCID_H_rgb_rev-20eebe74aef7d92e02732a18b6aa6bbb.svg" />);
    expect(getByTestId('header-image')).toHaveAttribute('src', 'https://developer.gov.bc.ca/static/BCID_H_rgb_rev-20eebe74aef7d92e02732a18b6aa6bbb.svg');
  });
});

describe('the component renders correctly', () => {
  it('should match the snapshot', () => {
    const tree = renderer
      .create(<HeaderImage />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
