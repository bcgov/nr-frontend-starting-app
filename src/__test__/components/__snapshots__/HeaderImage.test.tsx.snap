// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the component renders correctly should match the snapshot 1`] = `
<img
  className="sc-bczRLJ cywwia"
/>
`;
